shippingservice/README.md

```diff
@@ -0,0 +1,39 @@
+# Shipping Service
+
+The `Shipping Service` is a mock service where it returns a constant shipping cost if the cart is not empty. It also has the capability to generate a mock tracking number for the shipment. 
+
+```bal
+remote function GetQuote(stubs:GetQuoteRequest request) returns stubs:GetQuoteResponse {
+    log:printInfo(string `Received a quotation request with ${request.toString()}`);
+
+    stubs:CartItem[] items = request.items;
+    int count = 0;
+    foreach var {quantity} in items {
+        count += quantity;
+    }
+    float cost = 0.0;
+    if count != 0 {
+        cost = 8.99;
+    }
+    float cents = cost % 1;
+    int dollars = <int>(cost - cents);
+
+    return {
+        cost_usd: {currency_code: "USD", nanos: <int>(cents * FRACTION_SIZE), units: dollars}
+    };
+}
+
+remote function ShipOrder(stubs:ShipOrderRequest request) returns stubs:ShipOrderResponse {
+    log:printInfo(string `Received shipping order request with ${request.toString()}`);
+    stubs:Address address = request.address;
+    return {
+        tracking_id: generateTrackingId(string `${address.street_address}, ${address.city}, ${address.state}`)
+    };
+}
+```
+
+### A Glimpse of comparison
+
+The below figure provides a glimpse of the comparison of the source codes written in the Go and Ballerina languages.
+
+<img src="/images/shipping-service.png">
```

Comment:
I have added a code comparison image. Thoughts? We can do this for all the services if it is something interesting.

Reply:
@sameerajayasoma @xlight05 @dilanSachi @anupama-pathirage

Reply:
Guys, any comments on this? Shall we merge this if you guys are OK with the change.

Reply:
Sorry, Missed the notification, Ill look into this now

Reply:
I think, its better than having LOC. shows that the ballerina code is compact and more readable. 
I'm wondering if we should do this to highlight ballerina features as well? For the stuff we have in https://github.com/ballerina-guides/gcp-microservices-demo/blob/main/Highlights.md